CMakeLists.txt

```diff
@@ -0,0 +1,219 @@
+cmake_minimum_required(VERSION 3.0.2)
+set(CMP0048 NEW)
+
+project(_tango LANGUAGES CXX
+        VERSION 1.0)
+        
+#need to define the version of the library
+set(MAJOR_VERSION "9")
+set(MINOR_VERSION "2")
+set(PATCH_VERSION "6")
+
+#convenient versions
+set(LIBRARY_VERSION "${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}")
+if(NOT WIN32)
```

Comment:
This file is being added specifically for Windows builds, so is it really useful for other platforms?  If not useful, I suggest we just abort at this stage:
```
if(NOT WIN32)
message(FATAL_ERROR "This file is only targeting Windows builds - aborting!")
endif(NOT WIN32)
```